std/string.d

```diff
@@ -3010,6 +3107,13 @@ unittest
            [lineSep] ~ "hello world");
     assert(stripRight([paraSep] ~ "hello world" ~ paraSep) ==
            [paraSep] ~ "hello world");
+
+    assert(stripRight("     hello world     ", "x") ==
+           "     hello world     ");
+    assert(stripRight("     hello world     ", " ") ==
+           "     hello world");
+    assert(stripRight("     hello worldxy     ", "xy ") ==
+           "     hello world");
```

Comment:
Same goes for `stripRight`.  The original overload + unittests should be left as-is in their original order, and the new `stripRight` should be added *after* the original unittest, and the new tests should follow, in their own unittest block.